integration-test/utils/datafactory/schemas/mentorship.schema.ts

```diff
@@ -0,0 +1,34 @@
+import { pageSchema } from './page.schema';
+import { pageSectionSchema } from './pagesection.schema';
+import { feedbackSectionSchema } from './feedback.schema';
+
+export const mentorshipSchema = {
+  $ref: '#/definitions/mentorshipSchema',
+  definitions: {
+    mentorshipSchema: {
+      type: 'object',
+      properties: {
+        page: {
+          link: { ...pageSchema.definitions.pageSchema },
+        },
+        mentorSection: {
+          pageSection: {
+            link: { ...pageSectionSchema.definitions.pageSectionSchema },
```

Comment:
Why do you use `link` as a property? There should be `pageSection` and then reference to the schema.